extension/src/util/array.ts

```diff
@@ -48,3 +48,25 @@ export const splitMatchedOrdered = <T>(values: T[], existingOrder: T[]) => {
 
 export const reorderListSubset = <T>(subset: T[], supersetOrder: T[]): T[] =>
   splitMatchedOrdered(subset, supersetOrder)[0]
+
+export const performOrderedUpdate = (
+  order: string[],
```

Comment:
Didn't make sense to move `performSimpleOrderedUpdate` and not this one as well.